### ISSUE TITLE
High CPU load while different blocks are retried to index over and over again...

### ISSUE THREAD

**Issue by Kankalius** [blockscout/blockscout#8020]
Hi everyone, 
BlockScout has issues indexing a bunch of blocks with the following error message. It is always the same message for severa ldifferent block numbers: 

The errors we get over and over agaIn are these: 
2023-07-24T11:29:41.461 application=indexer fetcher=block_catchup first_block_number=6356365 last_block_number=6356365 step=import [error] failed to validate: [#Ecto.Changeset<action: nil, changes: %{}, errors: [hash: {"is invalid", [type: Explorer.Chain.Hash.Address, validation: :cast]}], data: #Explorer.Chain.Address<>, valid?: false>, #Ecto.Changeset<action: nil, changes: %{block_number: 6356365}, errors: [address_hash: {"is invalid", [type: Explorer>

This also brings up a high CPU load that only drops if we stop the BlockScout Service. 

Is there a way to limit how often indexing retries until it ignores it for good? Or can you help us, what exactly causes that error and how we can fix it to get these blocs indexed? 

### Environment

* Deployment type (Manual/Docker/Docker-compose): manual
* Elixir & Erlang/OTP versions (`elixir -version`): Elixir 1.13.0 compiled with Erlang/OTP 24 - erts-12.3
* Node JS version (`node -v`): v18.16.1
* Operating System: Ubuntu 20.04 LTS Server
* Blockscout Version/branch/commit: master - 5.2.0 beta
* Archive node type && version (Erigon/Geth/Nethermind/Ganache/?): polygon-edge 0.63

### Steps to reproduce

No special steps needed. The BlockSCout indexer keeps giving this error over and over again.

### Expected behaviour

Either index these blocks or ignore the failure after a few attempts. Would be nice to configure how many attempts it makes.

### Actual behaviour

The error shown above keeps being logged over and over again endlessly for several blocks.

**Comment by haidarabdillah** [comment]
any solve about this?

**Comment by Nexsuperne101** [comment]
no, unfortunately not.  CPU loading stayed at 85-90% until the block explorer was recently reinstalled. It is currently indexing (22% currently). I don't expect it to complete that for several weeks, as it has another 8 million or so blocks to get through. The error logs were dumped to the ether, as they were chewing up several GB of drive space every hour, requiring a manual log clear every few hours.

**Comment by vbaranov** [comment]
Please try to reproduce with the latest version of the backend.
